nss-fumbling-penguins/nutshell
# Rm friends list popup

## Link to Ticket
Closes # [Link]()

## Description of Proposed Changes

made friends list a popup from the navbar

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout rm-friends-list-popup

```
1. load page
1. check that show friends list button will hide and show friends list popup

## Impacted Areas in Application

List general components of the application that this PR will affect:

nav bar
friends list

## Mentions @username

Tag users that need to review this code

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
